README.md

```diff
@@ -387,14 +387,17 @@ Installation Instructions
   > **Note:**
   Defining the variable `scale_version` is optional for `scale_install_localpkg_path` and `scale_install_directory_pkg_path` installation methods. It is mandatory for `scale_install_repository_url` and `scale_install_remotepkg_path` installation methods. Furthermore, you'll need to configure an installation method
   by defining *one* of the following variables:
-  - `scale_install_repository_url` (eg: http://infraserv/gpfs_rpms/)
+  - `scale_install_repository_url` (eg: http://infraserv/scale/) - root of the Scale package folders and remember the last slash `/` in the url.
   - `scale_install_remotepkg_path` (accessible on Ansible managed node)
   - `scale_install_localpkg_path` (accessible on Ansible control machine)
   - `scale_install_directory_pkg_path` (eg: /opt/IBM/spectrum_scale_packages)
 
   > **Important:**
   If you are using the single directory installation method (`scale_install_directory_pkg_path`), you need to keep all required Spectrum Scale RPMs in a single user-provided directory.
 
+  - When using the `scale_install_repository_url` the other Ansible Roles will use the main path. example GUI will add /gpfs_rpms/` to the path and create seperate repo.
```

Comment:
Is there a way to customize the suffix, e.g. `.../myGpfsRpms/`? If that's the case then I'd suggest to say something like:

> By default, each role will add a suffix to this URL (e.g. `.../gpfs_rpms/`) which can be customized by adjusting the `xyz` variable.